CONTRIBUTING.md

```diff
@@ -1,46 +1,56 @@
+  - For code changes, add tests relevant to the fixed bug or new feature. 
+  - Pass the compile and tests - includes spell checks, formatting, etc.
+  - Commit header (first line) should convey what changed
+  - Commit body should include details such as why the changes are required and how the proposed changes
+  - DCO Signed 
+* If your PR is not getting reviewed or you need a specific person to review it, please reach out to the OpenEBS Contributors. See [OpenEBS Community](https://github.com/openebs/openebs/tree/master/community)
 
+## Sign your work
 
-### Sign your work
-
-We use the Developer Certificate of Origin (DCO) as a additional safeguard
-for the OpenEBS project. This is a well established and widely used
-mechanism to assure contributors have confirmed their right to license
-their contribution under the project's license.
-Please read [dcofile](https://github.com/openebs/openebs/blob/master/contribute/developer-certificate-of-origin).
-If you can certify it, then just add a line to every git commit message:
+We use the Developer Certificate of Origin (DCO) as a additional safeguard for the OpenEBS project. This is a well established and widely used mechanism to assure that contributors have confirmed their right to license their contribution under the project's license. Please read [dcofile](https://github.com/openebs/openebs/blob/master/contribute/developer-certificate-of-origin). If you can certify it, then just add a line to every git commit message:
 
 ````
   Signed-off-by: Random J Developer <random@developer.example.org>
 ````
 
-Use your real name (sorry, no pseudonyms or anonymous contributions).
-If you set your `user.name` and `user.email` git configs, you can sign your
-commit automatically with `git commit -s`. You can also use git [aliases](https://git-scm.com/book/tr/v2/Git-Basics-Git-Aliases)
-like `git config --global alias.ci 'commit -s'`. Now you can commit with
-`git ci` and the commit will be signed.
+Use your real name (sorry, no pseudonyms or anonymous contributions). The email id should match the email id provided in your GitHub profile. 
+If you set your `user.name` and `user.email` in git config, you can sign your commit automatically with `git commit -s`. 
+
+You can also use git [aliases](https://git-scm.com/book/tr/v2/Git-Basics-Git-Aliases) like `git config --global alias.ci 'commit -s'`. Now you can commit with `git ci` and the commit will be signed.
+
+## Setting up your Development Environment
+
+This project is implemented using Go and uses the standard golang tools for development and build. In addition, this project heavily relies on Docker and Kubernetes. It is expected that the contributors:
+- are familiar with working with Go
+- are familiar with Docker containers
+- are familiar with Kubernetes and have access to a Kubernetes cluster or Minikube to test the changes.
+
+For setting up Development environment on your local host, see the detailed instructions [here](./docs/developer.md).
```

Comment:
For setting up Development environment = For setting up a Development environment